src/OpenTelemetry.Instrumentation.AspNet/OpenTelemetry.Instrumentation.AspNet.csproj

```diff
@@ -6,10 +6,22 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Include="$(RepoRoot)\src\OpenTelemetry.Api\Internal\ExceptionExtensions.cs" Link="Includes\ExceptionExtensions.cs" />
+    <Compile Include="$(RepoRoot)\src\OpenTelemetry\Internal\ActivityInstrumentationHelper.cs" Link="Includes\ActivityInstrumentationHelper.cs" />
```

Comment:
@ejsmith Is this group of includes pretty much the same in each instrumentation csproj? If so, could we move to a props file and include that instead?

Reply:
They are slightly different in each, but there is probably a common base set.

Reply:
I've updated to move these includes to the Common.props file.